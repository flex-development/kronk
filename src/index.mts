/**
 * @file Package Entry Point
 * @module kronk
 */

export { default as keid } from '#enums/keid'
export { default as optionValueSource } from '#enums/option-value-source'
export type * from '#interfaces/index'
export * from '#lib/index'
export type * from '#types/index'
export * from '@flex-development/kronk/errors'
export * from '@flex-development/kronk/events'
export * from '@flex-development/kronk/parsers'
