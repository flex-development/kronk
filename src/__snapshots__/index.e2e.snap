// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:kronk > should expose public api 1`] = `
[
  "optionValueSource",
  "Argument",
  "Command",
  "Option",
  "VersionOption",
  "CommandError",
  "KronkError",
  "KronkEvent",
  "OptionEvent",
  "isArgument",
  "isCommandError",
  "isKronkError",
  "isKronkEvent",
  "isOption",
]
`;
